### ISSUE TITLE
Issue with `transit/long` encoding

### ISSUE THREAD

**Issue by jeremydw** [grow/grow#1155]
Issue encountered when building:

```
  File "/Users/jeremydw/.local/share/virtualenvs/project/lib/python3.9/site-packages/grow/common/utils.py", line 510, in slugify
    word = codecs.encode(word, 'translit/long')
```

We may want to find another way to slugify.

**Comment by Zoramite** [comment]
Which version of Grow is that in? I thought I updated the translit version in one of the recent updates since there was a problem with it on later versions of python.

**Comment by Zoramite** [comment]
Yeah, I think it was fixed in `1.0.1`. So you should update :) We are at `1.0.4` now.
